lib/AbsoluteGrid.jsx

```diff
@@ -60,6 +60,7 @@ export default class AbsoluteGrid extends React.Component {
       var gridItem = React.cloneElement(this.props.displayObject, _.assign(this.props.displayObject.props, {
         style: style,
         item: item,
+        itemsLength: this.props.items.length,
```

Comment:
You're right, I just ran a jsperf and it's identical. Going to go ahead and merge!